setup.md

```diff
@@ -4,166 +4,241 @@ This document will guide you through setting up Fennel on your
 * Lua version 5.1, 5.2, 5.3, or LuaJIT
 * LuaRocks or Git
 
-Note that Fennel can be used on non-Unix systems, but that is out of
-scope for this document.
+# Downloading and installing Fennel
 
-# Downloading Fennel
-
-Currently, you can either download Fennel from
-[https://github.com/bakpakin/Fennel](https://github.com/bakpakin/Fennel)
-or use LuaRocks to download Fennel. On certain operating systems, you
-might have Fennel available from your system package manager, but that is
-out of scope for this document.
+Downloading and installing Fennel on your system allows you to run
+Fennel scripts. Currently, you can download and install Fennel using
+Git or LuaRocks.
 
 Depending on which method you want to use, choose a subsection below:
 
 * [Using Git to download Fennel](#using-git-to-download-fennel)
 * [Using LuaRocks to download Fennel](#using-luarocks-to-download-fennel)
 
-**Tip**: If you are using an application that already has Fennel
-built-in (such as [TIC-80](https://tic.computer)) you might not need
-to download Fennel at all.
+**Tip**: If you are using software that supports Fennel, such as
+[TIC-80](https://tic.computer), you do not need to download Fennel,
+because you can use it inside of TIC-80.
+
+## Using Git to download and install Fennel
+
+Downloading and installing Fennel using Git allows you to use versions
+of Fennel that haven't been released yet and makes contributions to
+Fennel easier.
 
-## Using Git to download Fennel
+### To download Fennel
 
-You may want to use Git to install if:
+1. `cd` to a directory in which you want to download Fennel, such as
+   `~/src`
+2. Run `git clone https://github.com/bakpakin/Fennel`
 
-* You want to use a version that hasn't been released yet
-* You want to contribute changes to Fennel
-* You have Git installed and don't want to bother with another system
+### To install Fennel
 
-To install:
+1. Run `cd Fennel`
+2. Run `make fennel`
+3. Copy or link `fennel` to a location on your shell's `$PATH`, such
+   as `~/bin` or `/usr/local/bin`
 
-1. Change to the directory where you keep source checkouts (eg `~/src`)
-2. Run `git clone https://github.com/bakpakin/Fennel` in a shell
-3. Change directories with `cd Fennel`
-4. Run `make fennel` to compile Fennel into a standalone script
-5. Copy or link `fennel` to a location on your shell's `$PATH` (eg
-   `~/bin` or `/usr/local/bin`)
+**Note 1**: `Step 2.` above will compile Fennel into a standalone script
+called `fennel`.
 
-## Using LuaRocks to download Fennel
+**Note 2**: If the `fennel` script exists inside your `$PATH`
+directory, you can run `fennel filename.fnl` to run a Fennel file.
+
+## Using LuaRocks to download and install Fennel
 
 [LuaRocks](https://luarocks.org/) contains a repository of Lua
-packages. It allows you to download and install packages and automates
-installation steps.
+software packages. LuaRocks is convenient because it automates the
+downloading, installation, and uninstallation of software packages.
```

Comment:
"of Lua software packages" or just "of Lua software" maybe?

Reply:
I'll see what others decide on here, as I am not too familiar with LuaRocks in the first place :P